File: C++/1161.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int a,b;
    while(scanf("%d%d", &a, &b) != EOF){
        long long int totalA = 1, totalB = 1, total = 0;
        
        for(int i = 1; i <= a; i++){
            totalA *= i;
        }
        for(int y = 1; y <= b; y++){
            totalB *= y;
        }
        total = totalA + totalB;
        cout << total << "\n";
    }
    return 0; 
}

File: Java/1161.java
import java.util.ArrayList;
import java.util.Scanner;


public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while(sc.hasNextInt()){
            int a = sc.nextInt(), b = sc.nextInt();
            long totalA = 1, totalB = 1, total = 0;

            for(int i = 1; i <= a; i++){
                totalA = totalA * i;
            }

            for(int i = 1; i <= b; i++){
                totalB = totalB * i;
            }

            total = totalA + totalB;
            System.out.println(total);

        }
    }
}
